analyzers/fcc/Reader.py

```diff
@@ -74,7 +75,7 @@ class Reader(Analyzer):
     def process(self, event):
         store = event.input
 
-        def get_collection(class_object, coll_label, sort=True):
+        def get_collection(class_object, coll_label, coll_subtype=None, sort=True):
```

Comment:
please make sure that coll_subtype is specified when calling this with sort = False (check points of use for this function)

Reply:
Good point!